apidoc/Titanium/UI/Window.yml

```diff
@@ -528,7 +528,7 @@ properties:
   - name: hidesBackButton
     summary: Set this to true to hide the back button of navigation bar.
     description: |
-        When this property is set to true, the navigation controller hides its back button.
+        When this property is set to `true`, the navigation controller hides its back button.
```

Comment:
Still incorrect, see the quote: "navigation window" over "navigation controller". The latter one is not a known word in Titanium.